bahmnicore-omod/src/main/resources/liquibase.xml

```diff
@@ -4807,6 +4807,65 @@
             INSERT INTO role_privilege(role, privilege) VALUES ("Appointments:ManageAppointments", "Create Teleconsultation");
         </sql>
     </changeSet>
+    <changeSet id="bahmni-core-202303031445" author="Parvathy, Rahul">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="2">
+                SELECT count(*) FROM concept_set WHERE concept_id = 63 and concept_set = 62;
+            </sqlCheck>
+            <sqlCheck expectedResult="2">
+                SELECT count(*) FROM concept_set WHERE concept_id = 64 and concept_set = 62;
+            </sqlCheck>
+        </preConditions>
+        <comment>Remove dosing unit duplicates from concept set</comment>
+        <sql>
+            set @tablet_concept_set_id = 49;
```

Comment:
Not sure if this is the right approach, ID may be different and we have to find the ID using concept name.
@angshu @binduak your views?

Reply:
why can't we ingest this data using initializer?